aeon/utils/_testing/collection.py

```diff
@@ -1,21 +1,215 @@
 # -*- coding: utf-8 -*-
-"""Utility functions for generating panel data and learning task scenarios."""
+"""Utility functions for generating collections of time series."""
 # copyright: aeon developers, BSD-3-Clause License (see LICENSE file)
 
-__author__ = ["mloning", "fkiraly"]
+__author__ = ["mloning", "fkiraly", "TonyBagnall", "MatthewMiddlehurst"]
 __all__ = [
-    "make_classification_problem",
-    "make_regression_problem",
+    "make_3d_test_data",
+    "make_2d_test_data",
+    "make_unequal_length_test_data",
+    "make_nested_df_classification_data",
+    "make_nested_dataframe_regression_problem",
     "make_transformer_problem",
```

Comment:
```suggestion
    "make_3d_numpy_data",
    "make_2d_numpy_data",
    "make_numpy_list_data",
    "make_nested_df_classification_data",
    "make_nested_df_regression_data",
    "make_transformer_problem",
```

Naming could be a bit more descriptive. Nested DF one could be a single function with a parameter, probably.

Reply:
long names cause problems with text wrapping, but I'll have a go.

Reply:
hopefully sorted it out